settings/installer/ospd/ospd.spec

```diff
@@ -45,6 +45,9 @@ subparsers:
                     type: Value
                     help: Specifies whether ths SSL should be used for overcloud
                     default: 'no'
+                overcloud-hostname:
```

Comment:
Yes, it can be separated to different folder, we can use the feature we already have to pull settings from different folders and create setting folders for different roles/features: controllers, computes, nova, neutron or something like that.